code/game/objects/structures/icemoon/cave_entrance.dm

```diff
@@ -189,6 +188,6 @@ GLOBAL_LIST_INIT(ore_probability, list(
 		if(25)
 			new /mob/living/simple_animal/hostile/retaliate/clown/clownhulk(loc)
 		if(26)
-			new /obj/item/clothing/shoes/winterboots/ice_boots(loc)
-		if(27)
 			new /obj/item/book/granter/action/spell/sacredflame(loc)
+		if(27)
+			new /obj/item/organ/internal/cyberimp/arm/katana(loc)
```

Comment:
honestly this is really strong but i'm still gonna give it the greenlight for now